kura/org.eclipse.kura.util/src/main/java/org/eclipse/kura/util/base/ThrowableUtil.java

```diff
@@ -31,9 +32,11 @@ private ThrowableUtil() {
      *
      * @param throwable
      *            the throwable to propagate
+     * @throws RuntimeException
+     *             translates the provided exception as a runtime exception
      */
     public static void propagate(final Throwable throwable) {
-        throw new KuraRuntimeException(KuraErrorCode.INTERNAL_ERROR, throwable);
+        throw new RuntimeException(throwable);
```

Comment:
The method that I have added has a very good use case in Driver implementation for Kura Wires - soon to be available as a PR. The driver has to deal with all low-level complexities in which case it encounters several types of exceptions and based on specific exceptions, propagation is done but not for all. This could have been achieved using simply having multi-catch blocks but to save verbosity, this method is beneficial.